azure-pipelines.yml

```diff
@@ -37,7 +37,6 @@ jobs:
         PANDAS_VERSION: '*'
         TEST_DOCSTRINGS: 'true'
         JOBLIB_VERSION: '*'
-        CHECK_WARNINGS: 'true'
```

Comment:
Why this change?

Reply:
As a test be cause I was getting errors about FutureWarnings. I believe those tests should run only on cron based tasks because otherwise we get unrelated errors to PRs. What to you think?

Reply:
We are running those in scikit-learn. It allows to catch the warnings and ensure that you don't introduce new one by calling deprecated stuff. If this is in test, you can always filter a specific warning.

Reply:
So, what do you propose? I think that the main thing is not the deprecation and future warnings as errors. We should run those in periodic manner against the latest version of `scikit-learn` otherwise we are getting errors that are irrelevant to PRs. So, we could test against the latest (released) version of scikit-learn and even a previous one. The latest could be tested in schedule. Or something like that.